main.py

```diff
@@ -15,10 +16,15 @@
 OS_image="ubuntu"
 OS_image_version="latest"
 message = "testing message 123"
+path = 'C:\\Users\\aleks\\Desktop\\ContainerStudio\\outputs'
```

Comment:
We're running this project on Linux, not Windows

Reply:
I know, but it works the same in Windows and Linux. Path doesn't matter.